app/basicComponents/Input.js

```diff
@@ -31,6 +30,20 @@ const ActualInput = styled.input`
   outline: none;
 `;
 
+const ActualTextArea = styled.textarea`
```

Comment:
don't make such vast changes to widely used component in order to satisfy single insignificant use-case. Roll back the changes and use textArea instead